### ISSUE TITLE
Display Sync code expiry when showing the pairing codes

### ISSUE THREAD

**Issue by fmarier** [brave/brave-browser#31904]
Both the QR codes and the pairing words in SyncV2 expire after a little while. However, that's not immediately obvious to users, some of whom think it's some kind of backup that they should keep similar to a wallet recovery phrase or a TOTP QR code.

We should display a timer that shows the amount of time left for that code to make it obvious that it's a temporary code.

Maybe something like:

> This code is valid for the next **1:52:05.**

with the seconds counting down in real-time.

**Comment by aguscruiz** [comment]
Added the designs here:

![image](https://github.com/brave/brave-browser/assets/15734630/4c3ada13-9354-4ed6-8e8e-d43a050d3913)


### QR Code
https://www.figma.com/file/GXnHuBCVRY397UNJyvu44q/Desktop-Sync?type=design&node-id=307%3A5540&mode=design&t=jaHJy0eAsX76Wots-1

### Sync Code
https://www.figma.com/file/GXnHuBCVRY397UNJyvu44q/Desktop-Sync?type=design&node-id=307%3A3684&mode=design&t=jaHJy0eAsX76Wots-1

**Comment by fmarier** [comment]
This looks great. Thanks @aguscruiz!

What do you think we should show once the code has expired? Replace the words / QR corde with a red notice that the code is no longer valid?

**Comment by aguscruiz** [comment]
Yeah exactly. I added that into the design file aswell. Copy may need to be adjusted, let me know

**Comment by rebron** [comment]
@aguscruiz Can we get screens for Android and iOS as well? We probably need to reduce the size of the QR code for both Android and iOS to make some room.

**Comment by aguscruiz** [comment]
Added both here. 
https://www.figma.com/file/GXnHuBCVRY397UNJyvu44q/Sync-V2?type=design&node-id=318%3A1460&mode=design&t=IFaMKC3yvc3XYKo5-1

If there comes a point where we need vertical scroll, it would be fine. It'll probably on the small small screens. I mocked it up on the standard size and it seems to work ok.

I did make the QR code a tiny bit smalller
